_tools/TicketDescriptions.py

```diff
@@ -403,19 +439,32 @@ def translate_ncosmetic_desc(item, file_name):
     if item["tr_explain"] != "" and REDO_ALL == False:
         return -2
     
+    # Some items can use for all characters.
+    allcharacters = False
+    if "全キャラクターで選択可能になる。" in item["jp_explain"]:
+        allcharacters = True
+    
     # Some items are locked to one race and/or type.
     types = get_type_restrictions(item)
 
     # Some items hide your innerwear (these are mostly swimsuits).
     hideinner = False
     if "着用時はインナーが非表示になります。" in item["jp_explain"]:
         hideinner = True
+    
+    # Some items are not supported in the PSO2 blocks.
+    ngsonly = False
+    if "『PSO2』ブロック非対応" in item["jp_explain"]:
+        ngsonly = True
 
     # Translate the description.
-    item["tr_explain"] = (ndesc_formats[LANG] + "{typelock}").format(
+    item["tr_explain"] = ("{allcharacters}" + "{typelock}" + "{ngsonly}").format(
```

Comment:
The `{allcharacters}` token here needs to be renamed, otherwise python will be confused on line 463 when `allcharacters` means two different things.